src/thing/remove.test.ts

```diff
@@ -1064,6 +1065,230 @@ describe("removeDatetime", () => {
   });
 });
 
+describe("removeDate", () => {
+  it("removes the given date value for the given Predicate", () => {
+    const thingWithDate = getMockThingWithLiteralFor(
+      "https://some.vocab/predicate",
+      "1990-11-12",
+      "date"
+    );
+
+    const updatedThing = removeDate(
+      thingWithDate,
+      "https://some.vocab/predicate",
+      new Date(Date.UTC(1990, 10, 12))
+    );
+
+    expect(
+      updatedThing.predicates["https://some.vocab/predicate"].literals
+    ).toStrictEqual({
+      "http://www.w3.org/2001/XMLSchema#date": [],
+    });
+  });
+
+  it("accepts Properties as Named Nodes", () => {
+    const thingWithDate = getMockThingWithLiteralFor(
+      "https://some.vocab/predicate",
+      "1990-11-12",
+      "date"
+    );
+
+    const updatedThing = removeDate(
+      thingWithDate,
+      DataFactory.namedNode("https://some.vocab/predicate"),
+      new Date(Date.UTC(1990, 10, 12))
+    );
+
+    expect(
+      updatedThing.predicates["https://some.vocab/predicate"].literals
+    ).toStrictEqual({
+      "http://www.w3.org/2001/XMLSchema#date": [],
+    });
+  });
+
+  it("does not modify the input Thing", () => {
+    const thingWithDatetime = getMockThingWithLiteralFor(
```

Comment:
In this file too check for references to dateTime in your date tests :)

```suggestion
    const thingWithDate = getMockThingWithLiteralFor(
```